chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -136,18 +131,16 @@ class :math:`j` by the prediction.
     <https://arxiv.org/abs/1704.06857>`_. arXiv 2017.
 
     Args:
-        pred_labels (iterable of numpy.ndarray): A collection of predicted
-            labels. The shape of a label array
-            is :math:`(H, W)`. :math:`H` and :math:`W`
-            are height and width of the label.
-            For example, this is a list of labels
-            :obj:`[label_0, label_1, ...]`, where
-            :obj:`label_i.shape = (H_i, W_i)`.
-        gt_labels (iterable of numpy.ndarray): A collection of ground
-            truth labels. The shape of a ground truth label array is
-            :math:`(H, W)`, and its corresponding prediction label should
-            have the same shape.
-            A pixel with value :obj:`-1` will be ignored during evaluation.
+        pred_labels (iterable of numpy.ndarray): See the table below.
+        gt_labels (iterable of numpy.ndarray): See the table below.
+
+    .. csv-table::
+        :header: name, shape, dtype, format
+
+        :obj:`pred_labels`, ":math:`[(H, W)]`", :obj:`int32`, \
+        ":math:`[-1, \#class - 1]`"
```

Comment:
`[0, #class - 1]` ? `-1` stands for ignored class and the prediction should not contain it.

Reply:
Thank you